README.md

```diff
@@ -10,7 +10,16 @@ Crystal is a programming language with the following goals:
 * Have compile-time evaluation and generation of code, to avoid boilerplate code.
 * Compile to efficient native code.
 
-## Build
+## Contributing to These Docs
```

Comment:
I would vote for "language reference" and "api reference". They're very clear and specific.